src/autoscaler/cred_helper/loader.go

```diff
@@ -15,9 +16,23 @@ import (
 
 type PluginManager struct {
 	client *plugin.Client
+
+	// ctx is used to shutdown the device manager
+	ctx    context.Context
```

Comment:
do we use this context anywhere ?
we should probably  be using it in the RPC client requests ?

Reply:
OK ... so https://www.quora.com/What-is-the-difference-between-kill-command-and-interrupt-a-process
suggests that TERM should be caught and kill(9) should not be called.
So we should be calling sig term or `.Terminate()` and killwithFire should not be used (SEG_F/SIGKILL)